ClinGen/clincoded
# #2004, #2014, #2040, #2012, #2039: Add/Edit Affiliations

**Steps to Test #2004, #2014, #2012**
 Save, provision, approve and publish a variant as TP53, Lysosomal Storage Disorders, and Glaucoma

**Steps to Test #2040**


**Steps to Test #2039**
Choose the following affiliations to make sure they exist:
Dosage Sensitivity
Actionability
Somatic Cancer
ABCD1
Craniosynostosis
Orphan Diseases
Limb Girdle Muscular Dystrophy
